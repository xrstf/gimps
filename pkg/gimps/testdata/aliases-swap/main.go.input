package main

import (
	a "fmt"
	b "log"
)

func main() {
	a.Println("")
	b.Println("")
}
