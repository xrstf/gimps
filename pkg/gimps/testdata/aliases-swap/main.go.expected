package main

import (
	b "fmt"
	a "log"
)

func main() {
	b.Println("")
	a.Println("")
}
